script/setup/install-runhcs-shim

```diff
@@ -28,13 +29,15 @@ cleanup() {
 trap 'cleanup' EXIT
 
 export GOOS
-
-(
+if [ "$HCSSHIM_SRC" == "" ]
+then
     set -e -x
     cd "$tmpdir"
     git init .
     git remote add origin "$RUNHCS_REPO"
     git fetch --tags --depth=1 origin ${RUNHCS_VERSION}
-    git checkout "refs/tags/${RUNHCS_VERSION}" || git checkout "refs/heads/${RUNHCS_VERSION}" || git checkout "${RUNHCS_VERSION}"
-    GO111MODULE=on go build -mod=vendor -o "${DESTDIR}/containerd-shim-runhcs-v1.exe" ./cmd/containerd-shim-runhcs-v1
-)
\ No newline at end of file
+else
+    cd "${HCSSHIM_SRC}"
+fi
+git checkout "refs/tags/${RUNHCS_VERSION}" || git checkout "refs/heads/${RUNHCS_VERSION}" || git checkout "${RUNHCS_VERSION}"
+GO111MODULE=on go build -mod=vendor -o "${DESTDIR}/containerd-shim-runhcs-v1.exe" ./cmd/containerd-shim-runhcs-v1
```

Comment:
Would you remove the end-of-line?

Reply:
End of line from which line.  Last line don't have any end of line/ extra char

Reply:
I mean this one 
![image](https://user-images.githubusercontent.com/17510957/216001249-68bd1ffb-76c9-4e5d-8a7d-d6d05656ab56.png)

Reply:
Resolved